core/src/main/java/org/infinispan/configuration/parsing/Parser.java

```diff
@@ -553,6 +553,10 @@ private void parseContainer(XMLExtendedStreamReader reader, ConfigurationBuilder
                builder.shutdown().hookBehavior(ShutdownHookBehavior.valueOf(value));
                break;
             }
+            case ZERO_CAPACITY_NODE: {
+               builder.zeroCapacityNode(Boolean.valueOf(value));
```

Comment:
```suggestion
               builder.zeroCapacityNode(Boolean.parseBoolean(value));
```

Reply:
I was using valueOf as for the statistics, but parseBoolean is the same